main/webapp/modules/core/langs/translation-en.json

```diff
@@ -343,12 +344,12 @@
     "core-project/remove-all": "Remove all facets",
     "core-project/export-project": "Export project",
     "core-project/project-data-package": "Project data package",
-    "core-project/tab-value": "Tab-separated value",
-    "core-project/comma-sep": "Comma-separated value",
-    "core-project/html-table": "HTML table",
-    "core-project/excel": "Excel (.xls)",
-    "core-project/excel-xml": "Excel 2007+ (.xlsx)",
-    "core-project/odf": "ODF spreadsheet",
+    "core-project/tab-value": "Tab-separated value file",
+    "core-project/comma-sep": "Comma-separated value file",
+    "core-project/html-table": "HTML table file",
+    "core-project/excel": "Excel (.xls) file",
+    "core-project/excel-xml": "Excel 2007+ (.xlsx) file",
+    "core-project/odf": "ODF spreadsheet file",
```

Comment:
Is it really worth adding "file" at the end of all these labels? For instance, "ODF spreadsheet file" sounds quite redundant for me. I would keep these labels as they were.

Reply:
> Is it really worth adding "file" at the end of all these labels? For instance, "ODF spreadsheet file" sounds quite redundant for me. I would keep these labels as they were.

   Also, spreadsheet file is not redundant: the spreadsheet can be saved to the cloud or elsewhere. But in my proposition, the menu item is precise, it tells you when you will generate files.